<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDock</class>
 <widget class="QDockWidget" name="FindReplaceDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find/Replace</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFind">
        <property name="text">
         <string>Find:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editFind"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonNext">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/icons/scalable/down.svg</normaloff>:/icons/scalable/down.svg</iconset>
        </property>
        <property name="shortcut">
         <string>F3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="buttonPrevious">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/icons/scalable/up.svg</normaloff>:/icons/scalable/up.svg</iconset>
        </property>
        <property name="shortcut">
         <string>Shift+F3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelReplace">
        <property name="text">
         <string>Replace:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editReplace"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonReplace">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="buttonReplaceAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Replace all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
      <item>
       <widget class="QLabel" name="labelMessage">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSearchType">
        <property name="toolTip">
         <string>Search in element type instead of value.</string>
        </property>
        <property name="text">
         <string>Search Type</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxRegExp">
        <property name="text">
         <string>Interpret as RegExp</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxMatchCase">
        <property name="text">
         <string>Match Case</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDock</receiver>
   <slot>clickedNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPrevious</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDock</receiver>
   <slot>clickedPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReplace</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDock</receiver>
   <slot>clickedReplace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReplaceAll</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDock</receiver>
   <slot>clickedReplaceAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedNext()</slot>
  <slot>clickedPrevious()</slot>
  <slot>clickedReplace()</slot>
  <slot>clickedReplaceAll()</slot>
 </slots>
</ui>
